src/modules/Ticket/index.tsx

```diff
@@ -259,30 +301,69 @@ function Ticket() {
                 name='resolver_id'
                 required={true}
                 label={'Resolver'}
-                value={formik.values.resolver_id}
+                value={formik.values.resolver_id!}
                 options={userOptions}
                 onChange={formik.handleChange}
                 onBlur={formik.handleBlur}
                 error={formik.touched.resolver_id && formik.errors.resolver_id}
               />
             </FormControl>
+            {formik.values.asset_url?.map((item, index) => (
+              <Box
+                display={'flex'}
+                alignItems={'center'}
+                gap={3}
+                pl={3}
+                pr={2}
+                py={1}
+                border={1}
+                borderColor={'grey.400'}
+                borderRadius={1}
+              >
+                <Typography
+                  variant='body2'
+                  flex={1}
+                  overflow={'hidden'}
+                  textOverflow={'ellipsis'}
+                  whiteSpace={'nowrap'}
+                  mr={'auto'}
+                >
+                  {item}
+                </Typography>
+                <IconButton
+                  onClick={() => {
+                    handleUpdateAsset(
+                      formik.setFieldValue,
+                      index,
+                      formik.values.asset_url!
+                    );
+                  }}
+                  sx={{ p: 1 }}
+                >
+                  <CloseRounded sx={{ fontSize: 16 }} />
+                </IconButton>
+              </Box>
+            ))}
+            <UploadBucket
+              isLoading={isLoading}
+              file={file}
+              value={formik.values.asset_url!}
+              name={'asset_url'}
+              handleChange={handleChangeFile}
+              removeFile={removeFile}
+            />
           </Box>
-          <UploadBucket
-            isLoading={isLoading}
-            file={file}
-            value={formik.values.asset_url}
-            name={'asset_url'}
-            handleChange={handleChangeFile}
-            removeFile={removeFile}
-          />
-          <Box sx={{ display: 'flex', justifyContent: 'end', pt: 3 }}>
-            <Button isLoading={creatingTicket} type='submit'>
-              Create
-            </Button>
-          </Box>
-        </Box>
-      </Paper>
-    </Box>
+        </DialogContent>
+        <DialogActions>
+          <Button variant='text' onClick={() => setOpen(false)}>
+            Cancel
+          </Button>
+          <Button isLoading={creatingTicket} type='submit'>
+            {!isEdit ? 'Create' : 'Update'}
+          </Button>
+        </DialogActions>
+      </Box>
+    </Dialog>
   );
 }
 export default Ticket;
```

Comment:
Wrap the Ticket in React.memo as we want to avoid unecessary re-rendering of ticket